src/nodestore.rs

```diff
@@ -1,3 +1,5 @@
+use crate::{Lat, Lon};
```

Comment:
There wasn't any documentation for this class, but I gather it's to be used when hydrating a Way/Reference's inner Node references with actual locations. Right?

I did test that it works when used in this capacity after these changes.

Reply:
I can't remember for sure now. It's the early stages of a "store node locations". You can ignore this file.

Reply:
I'm actually quite interested in this functionality. I'm going to open an issue to start a discussion since I think there's a few tradeoffs.